capellambse_context_diagrams/collectors/tree_view.py

```diff
@@ -51,24 +52,21 @@ def process_class(self, cls: ClassInfo, params: dict[str, t.Any]):
             assert cls.prop is not None
             self._process_box(cls.target, cls.partition, params)
 
-            aggregation_kind_excludes = {
-                None,
-                modeltypes.AggregationKind.UNSET,
-            }
             if cls.prop.association is not None:
                 edge_id = cls.prop.association.uuid
             else:
                 logger.warning(
                     "No Association found for %s set on 'navigable_members'",
                     cls.prop._short_repr_(),
                 )
-                if cls.prop.kind not in aggregation_kind_excludes:
+                if cls.prop.kind != modeltypes.AggregationKind.UNSET:
                     styleclass = cls.prop.kind.name.capitalize()
                 else:
                     styleclass = "Association"
 
-                i = self._edge_count.setdefault(cls.prop.uuid, 0)
-                i += 1
+                self._edge_count.setdefault(cls.prop.uuid, 0)
+                self._edge_count[cls.prop.uuid] += 1
```

Comment:
The beauty of a defaultdict is that you don't need the explicit `setdefault` call anymore. 🙃 But having it this way doesn't exactly hurt either.